tensorflow_lite_support/c/task/vision/image_segmenter.h

```diff
@@ -46,11 +46,11 @@ limitations under the License.
 /// options.base_options.compute_settings.cpu_settings.num_threads = 3
 ///
 /// Create TfLiteImageSegmenter using the options:
-/// If error information is not needed in case of failure:
+/// If error information is not nedded in case of failure:
```

Comment:
nit: typo.

@priankakariatyml 
It looks like you reverted several typo fixes that I added in the previous pull request. Please correct them. Thanks!

Reply:
Sorry. Had some conflicts after committing some formatting changes and resolved with mine. It's been reverted back.